app/models/task.py

```diff
@@ -1,5 +1,31 @@
 from app import db
-
+from flask import make_response
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
```

Comment:
The default for a nullable constraint is True, so you can absolutely leave that out here if you would like!